Tools/ardupilotwaf/boards.py

```diff
@@ -276,6 +277,11 @@ class bebop(linux):
 
     def configure_env(self, cfg, env):
         super(bebop, self).configure_env(cfg, env)
+        os.environ["PKG_CONFIG_PATH"] = '/usr/lib/arm-linux-gnueabihf/pkgconfig'
```

Comment:
I don't like the use of a fixed path. I think we should define the PKGCONFIG var like we do for AR (https://github.com/ArduPilot/ardupilot/blob/master/Tools/ardupilotwaf/toolchain.py#L127), let Waf find the right pkgconfig and it would look in the correct paths. Then, in future work, we could provide the lib in our own version of the tools (which already exist and are downloaded by the CI systems).